macos.md

```diff
@@ -191,9 +191,8 @@ Make sure that you're running the 2nd-newest macOS version or the newest version
 
     ```bash
     [[ -d /opt/homebrew/var/postgresql@16 ]] && PGDATA_TMP=/opt/homebrew/var/postgresql@16 || PGDATA_TMP=/usr/local/var/postgresql@16
-    echo "\nexport PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'`
+    echo -e "export PGDATA=$PGDATA_TMP\nexport PSQL_PAGER=\"less --chop-long-lines --header 1\"\nexport LANG=en_US.UTF-8" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshrc' || echo '.bash_profile'`
```

Comment:
why do we need the -e flag now, and why didn't we need it before?

Reply:
Echo doesn't handle new lines correctly on `Windows` and `Linux`

```bash
echo "\n" vs echo -e "\n"
```
<img width="1920" alt="Screenshot 2024-08-07 at 10 42 16" src="https://github.com/user-attachments/assets/25db5f99-2eb0-4cc5-b7e7-7259078cec95">

![Screenshot 2024-08-07 104017](https://github.com/user-attachments/assets/0d5cbd11-cc82-43a3-a802-ea464437801f)